l1-contracts/contracts/upgrades/GatewayHelper.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {IL1SharedBridgeLegacy} from "../bridge/interfaces/IL1SharedBridgeLegacy.sol";
+import {IBridgehub} from "../bridgehub/IBridgehub.sol";
+
+import {ZKChainSpecificForceDeploymentsData} from "../state-transition/l2-deps/IL2GenesisUpgrade.sol";
+
+import {ZKChainStorage} from "../state-transition/chain-deps/ZKChainStorage.sol";
+
+library GatewayHelper {
+    function getZKChainSpecificForceDeploymentsData(ZKChainStorage storage s) internal view returns (bytes memory) {
+        address sharedBridge = IBridgehub(s.bridgehub).sharedBridge();
```

Comment:
I assume this is before/during the upgrade, so it will have to stay sharedBridge?

Reply:
yes